ddtrace/internal/telemetry/writer.py

```diff
@@ -218,6 +221,16 @@ def __init__(self):
         # type: () -> None
         super(TelemetryMetricsWriter, self).__init__(interval=_get_telemetry_metrics_interval_or_default())
         self._namespace = MetricNamespace()
+        self._logs = []  # type: List[LogMetric]
+
+    def log_metric(self, level, message, stack_trace="", tags={}):
+        # type: (str, str, str, MetricTagType) -> None
+        """
+        Queues log metric
+        """
+        if config._telemetry_metrics_enabled:
+            with self._lock:
+                self._logs.append(LogMetric(level, message, stack_trace, tags))
```

Comment:
I am not sure if we need a LogMetric class. We added a class for Metrics because we needed to aggregate metrics. Logs can be formatted into a dict directly. This will better align with other events and comes with some performance improvements. 

```suggestion
                self._logs.append(
                  {
                      "message": message,
                      "level": level.upper(),
                      "tracer_time": int(self.tracer_time),
                      "statck_trace": stack_trace,
                      "tags": ",".join(["%s:%s" % (k, v) for k, v in tags.items()]
                  }
                )
```